edx_solutions_api_integration/courses/views.py

```diff
@@ -1847,7 +1853,13 @@ def get(self, request, course_id):  # pylint: disable=W0613
         if cached_enrollments_data and not len(request.query_params):
             enrollment_count = cached_enrollments_data.get('enrollment_count')
         else:
-            users_enrolled_qs = CourseEnrollment.objects.users_enrolled_in(course_key).exclude(id__in=exclude_users)
+            if exclude_type:
+                exclude_roles = [CourseInstructorRole.ROLE, CourseStaffRole.ROLE,
```

Comment:
@naeem91 thanks for reviewing. Actually, we need to exclude four roles to make enrolled users count consistent on different pages as the original requirement was to exclude these users from participant count. But the previous call just excludes one role `[CourseObserverRole.ROLE]` which is correct and uses to get `users_passed`,  `avg_progress` and `avg_grade` metrics, etc but not in case of enrolled users count. So I excluded and cached these roles separately, rather than updating the previous one which is using to get different metrics.

Reply:
`get_aggregate_exclusion_user_ids` is picking roles from [settings](https://github.com/edx-solutions/api-integration/blob/1fbd82f37e9bbf042414bc4a1f254a55484b6a38/edx_solutions_api_integration/utils.py#L195).

And on QA/INT I can see it is already set to exclude four roles:

<img width="571" alt="Screen Shot 2020-04-21 at 12 13 50 PM" src="https://user-images.githubusercontent.com/6681794/79837032-4ad00280-83ca-11ea-9ef1-e6d70c7e9a12.png">

Reply:
got it. taking this specific change back.